cmd/bundle/generate/job.go

```diff
@@ -0,0 +1,117 @@
+package generate
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/bundle/config/generate"
+	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/databricks/cli/libs/dyn"
+	"github.com/databricks/cli/libs/dyn/convert"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+	"github.com/spf13/cobra"
+)
+
+func NewGenerateJobCommand() *cobra.Command {
+	var outputDir string
+	var force bool
+
+	cmd := &cobra.Command{
+		Use:     "job JOB_ID",
```

Comment:
I think @lennartkats-db 's earlier comment makes sense, where we use `--existing-job-id` as a flag to specify what to generate from, instead of a mandatory positional arg.

Then we can expand into generating jobs without there being a reference.

Reply:
Indeed, thanks, done